src/Microsoft.AspNet.Routing/RouteBase.cs

```diff
@@ -105,13 +106,9 @@ public virtual Task RouteAsync(RouteContext context)
             {
                 return TaskCache.CompletedTask;
             }
+            _logger.MatchedRouteName(Name, ParsedTemplate.TemplateText);
```

Comment:
This isn't right, it's `MatchedRoute`. We're just logging the name as part of teh dataz